.github/workflows/ci-tests.yml

```diff
@@ -243,20 +243,13 @@ jobs:
       # working dir on D drive which causes problems.
       run: |
         mkdir temp_pytest
-        python -m tox -e winpy -- -n 2 test
+        python -m tox -e winpy -- --cov=sqlfluff -n 2 test
     - name: Upload Coverage Report
       uses: codecov/codecov-action@v3
       with:
         # Public token to allow coverage from forks to work.
         token: fd44b020-c716-4f72-b5f3-03fa20119956
         files: ./coverage.xml
-    - name: Coveralls Parallel
```

Comment:
Removed this because I've confirmed we're hitting 100% coverage without running Windows tests. Removing this means we see coverage results **faster** on PR builds, because the Windows build almost always finishes after the Linux one.

Reply:
Sounds sensible. We can always bring it back if we end up with more windows specific code.